### ISSUE TITLE
BufNNDSVD does nothing with `activations` buffer

### ISSUE THREAD

**Issue by tedmoore** [flucoma/flucoma-sc#112]
Code to demonstrate:
```
(
~src = Buffer.read(s,FluidFilesPath("Nicol-LoopE-M.wav"));
~bases = Buffer.new(s);
~activations = Buffer.new(s);
~resynth = Buffer.new(s);
)

//how many bases do I need to decompose the buffer with 90% accuracy
(
FluidBufNNDSVD.processBlocking(s, ~src, ~bases, ~activations, coverage: 0.5, method: 1,action:{
	"% bases".format(~bases.numChannels).postln;
	{
		~bases.plot;
		~activations.plot;
	}.defer;
});
)
```

This throws the warning that `WARNING: Buffer not allocated, can't plot data` which is in reference to the `activations` buffer. It makes sense to me that the activations buffer is not filled, but why then to we require (it is *required*) the user to pass a buffer?

Can we hide this?

**Comment by weefuzzy** [comment]
Checking in Max reveals that the activations buffer is filled after all, so we're able to narrow this down to an SC problem. Which is that the SC class isn't `query`ing the activations buffer upon completion. Doing this manually reveals that all is well

**Comment by weefuzzy** [comment]
fixed by 2996f7cd17f817f528b69573fd51798c33711871
